src/plugins/intel_gpu/tests/unit/test_cases/scatter_elements_update_gpu_test.cpp

```diff
@@ -390,14 +522,109 @@ INSTANTIATE_TEST_SUITE_P(scatter_elements_update_gpu_formats_test_f32_4d,
+                                 ::testing::ValuesIn({true, false}),
+                                 ::testing::Values(format::bfyx)
+                         ),
+                         PrintToStringParamName());
+
+INSTANTIATE_TEST_SUITE_P(scatter_elements_update_gpu_reduction_test_f32_3d,
+                         scatter_elements_update_gpu_reduction_test_f32,
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(generateScatterElementsUpdateParams3D<float, int32_t>()),
+                                 ::testing::ValuesIn(reduce_modes),
+                                 ::testing::ValuesIn({true, false}),
+                                 ::testing::Values(format::bfzyx)
+                         ),
+                         PrintToStringParamName());
+
+INSTANTIATE_TEST_SUITE_P(scatter_elements_update_gpu_reduction_test_f32_4d,
+                         scatter_elements_update_gpu_reduction_test_f32,
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(generateScatterElementsUpdateParams4D<float, int32_t>()),
+                                 ::testing::ValuesIn(reduce_modes),
+                                 ::testing::ValuesIn({true, false}),
+                                 ::testing::Values(format::bfwzyx)
+                         ),
+                         PrintToStringParamName());
+
+INSTANTIATE_TEST_SUITE_P(scatter_elements_update_gpu_reduction_none_test_f32_2d,
+                         scatter_elements_update_gpu_reduction_test_f32,
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(generateScatterElementsUpdateParams2D<float, int32_t>()),
+                                 ::testing::Values(ov::op::v12::ScatterElementsUpdate::Reduction::NONE),
+                                 ::testing::Values(true),
+                                 ::testing::Values(format::bfyx)
+                         ),
+                         PrintToStringParamName());
+
+INSTANTIATE_TEST_SUITE_P(scatter_elements_update_gpu_max_reduction_use_init,
+                         scatter_elements_update_gpu_reduction_test_f32,
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(generateScatterElementsUpdateParams2D<float, int32_t>()),
+                                 ::testing::Values(ov::op::v12::ScatterElementsUpdate::Reduction::MAX),
+                                 ::testing::Values(true),
+                                 ::testing::Values(format::bfyx)
+                         ),
+                         PrintToStringParamName());
+
+// Disabled due to bug in reference implementation - see function reduction_neutral_value()
+// in core/reference/include/ngraph/runtime/reference/scatter_elements_update.hpp.
+// For MAX reduction it returns numeric_limits<T>::min() which is minimal *positive*, not truly minimal value of type T,
```

Comment:
I tested and numeric_limits<int>::min() returned -2147483648. Why is it bug?

Reply:
_> I tested and numeric_limits<>::min() returned -2147483648. Why is it bug?_
I assume you've tested numeric_limits<**int**>::min(). 
It works perfectly fine for integral types, but for **float**  it would return smth like  1.0e-38 - i.e., positive value (see https://en.cppreference.com/w/cpp/types/numeric_limits/min).